master/logging.lua

```diff
@@ -0,0 +1,17 @@
+local util = require "master.util"
+
+local state = util.mesos_get_state()
+if state == nil then
+    ngx.status = ngx.HTTP_SERVICE_UNAVAILABLE
+    ngx.say("503 Service Unavailable: invalid Mesos state.")
+    return ngx.exit(ngx.HTTP_SERVICE_UNAVAILABLE)
+end
+for _, agent in ipairs(state["slaves"]) do
+    if agent["id"] == ngx.var.agentid then
```

Comment:
I diffed logging.lua with agent.lua and:
```
 for _, agent in ipairs(state["slaves"]) do
     if agent["id"] == ngx.var.agentid then
-        local split_pid = agent["pid"]:split("@")
-        ngx.var.agentaddr = split_pid[2]
-        ngx.log(
-            ngx.DEBUG, "agentid / agentaddr:" ..
-            ngx.var.agentid .. " / " .. ngx.var.agentaddr
-            )
+        ngx.var.agentaddr = agent["hostname"]
         return
     end
 end
```

Could you please give me some context on this ?